components/TextFields/src/ContainedInputView/MDCBaseTextField.m

```diff
@@ -79,6 +79,7 @@ - (void)commonMDCBaseTextFieldInit {
 #pragma mark View Setup
 
 - (void)initializeProperties {
+  self.font = [self uiTextFieldDefaultFont];
```

Comment:
If this means that the font should be non nil after we initialize it lets make a unit test for it.